.github/workflows/links-test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           subfolder: /github/workspace/
 
           # A comma-separated list of file types to cover in the URL checks
-          file_types: .md
+          file_types: .md, .mdx
```

Comment:
I think we need to remove `space` after `,` so as to make that valid.

Reply:
It did not complain 😃 .

Anyways, It is easier to change than to test if it works both ways.

Reply:
Thanks, actually i find in logs for '.mdx' files, it was not showing earlier, now it shows.

Thank you